lib/handlers/app/handlers/getFullTheme.js

```diff
@@ -0,0 +1,30 @@
+var forms = require('fh-forms');
+var logger = require('../../../util/logger').getLogger();
+
+/**
+ * Get A Theme Associated With A Project
+ * @param {object}   req    Request Object
+ * @param {object}   res    Response Object
+ * @param {function} next   Middleware Next Function
+ */
+module.exports = function getFullTheme(req, res, next) {
+  var params = {
+    appId: req.params.projectid || req.params.id
```

Comment:
This can just be `req.params.projectid` as that is defined on the router that is being used.